### PR TITLE
COR-573 encode individual list options to URL

### PR DESCRIPTION
**Pull request by ludydoo**
This PR adds the functionality of converting the new ListIndividualOptions fields to a URL representation.

### CODE REVIEW

File: internal/api/individual_list_options_test.go [hunk 1]
```diff
@@ -287,6 +279,194 @@ func TestListIndividualsOptions_QueryParams(t *testing.T) {
...
+			o:    ListIndividualsOptions{CountryID: countryId, PrefersToRemainAnonymous: pointers.Bool(true)},
+			want: "/countries/usa/individuals?prefers_to_remain_anonymous=true",
+		}, {
+			name: "prefersToRemainAnonymous (false)",
+			o:    ListIndividualsOptions{CountryID: countryId, PrefersToRemainAnonymous: pointers.Bool(false)},
+			want: "/countries/usa/individuals?prefers_to_remain_anonymous=false",
+		}, {
+			name: "presentsProtectionConcerns",
+			o:    ListIndividualsOptions{CountryID: countryId, PresentsProtectionConcerns: pointers.Bool(true)},
+			want: "/countries/usa/individuals?presents_protection_concerns=true",
```
Comment (internetti) [inline internal/api/individual_list_options_test.go:437]: actually, when it comes to these search parameters, we could use abbreviations, I think. only in the url though, would still keep the long versions internally. if that doesn't overcomplicate things

Comment (ludydoo) [reply internal/api/individual_list_options_test.go:437]: Perhaps we can put this as an improvement for the future ?

Comment (internetti) [reply internal/api/individual_list_options_test.go:437]: if this is complicated, we can do it later. but do you agree with the idea?

Comment (ludydoo) [reply internal/api/individual_list_options_test.go:437]: Yeah, I think that large searches definitely can oeverflow the max length for a URL, so shortening those keys make sense.

Comment (internetti) [reply internal/api/individual_list_options_test.go:437]: okay, I'll create a ticket

**Review by internetti — APPROVED** [review]

**Pull request merged**